README.md

```diff
@@ -68,12 +68,11 @@ The easiest way to install Spyder is:
 
 ### On Windows:
 
-- Or using one of these scientific Python distributions:
+- Using one of our [executable installers](https://github.com/spyder-ide/spyder/releases)
+- Or installing one of these scientific Python distributions:
   1. [Anaconda](http://continuum.io/downloads)
   2. [WinPython](https://winpython.github.io/)
-  3. [Python(x,y)](http://pythonxy.googlecode.com)
-- Using one of our executable installers, which can be found
-  [here](https://github.com/spyder-ide/spyder/releases).
+  3. [Python(x,y)](http://python-xy.github.io/)
```

Comment:
I see that those distributions are missing Spyder 3.x, so should we drop them until they add support for 3rd version?